backend/server/project.go

```diff
@@ -692,6 +692,11 @@ func (s *Server) registerProjectRoutes(g *echo.Group) {
 	})
 
 	g.POST("/project/:projectID/sync-sheet", func(c echo.Context) error {
+		// TODO(tianzhou): uncomment this after adding the test harness to using Enterprise version.
```

Comment:
comment out this for now. This PR is already large, will fix the Enterprise license test harness in a followup PR.